### PR TITLE
[YUNIKORN-2217] Add Missing REST Endpoint description for batch event in documentation

### PR DESCRIPTION
**Pull request by Haser0305**
### What is this PR for?
Adds documentation for the /ws/v1/events/batch API endpoint. 

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2217

### CODE REVIEW

**Review by wilfred-s — APPROVED** [review]
LGTM, waiting for a review from @pbacsko before committing to make sure we have covered the latest changes

**Review by pbacsko — CHANGES_REQUESTED** [review]
Also document the URL query parameters:
* `count`: maximum number of events in the response
* `start`: the start ID of the events

eg. `localhost:9080/ws/v1/events/batch?start=200&count=1000`

If the ID is not in the ring buffer (too low or too high), then the response contains the current lowest and highest ID in the response with `EventRecords` being empty.

See
https://github.com/apache/yunikorn-core/blob/681a4cbf4b9929e579674e44e6309acf0e728e29/pkg/events/event_system.go#L57-L61

https://github.com/apache/yunikorn-core/blob/681a4cbf4b9929e579674e44e6309acf0e728e29/pkg/webservice/handlers.go#L1055-L1082

**Review by pbacsko — APPROVED** [review]
+1

**Pull request closed without merging**
